test/parallel_api/ranges/std_ranges_test.h

```diff
@@ -190,53 +192,50 @@ struct test
     {
         static_assert(mode == data_in_out || mode == data_in_out_lim);
 
-        constexpr int max_n = 10;
-        DataType data_in[max_n] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
-        DataType data_out[max_n] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
-        DataType expected[max_n] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+        Container cont_in(exec, n_in, [](auto i) { return i;});
+        Container cont_out(exec, n_out, [](auto i) { return 0;});
+        Container cont_exp(exec, n_out, [](auto i) { return 0;});
 
         assert(n_in <= max_n);
         assert(n_out <= max_n);
 
-        auto src_view = tr_in(std::ranges::subrange(data_in, data_in + n_in));
-        auto expected_res = checker(src_view, std::ranges::subrange(expected, expected + n_out), args...);
-        {
-            Container cont_in(exec, data_in, n_in);
-            Container cont_out(exec, data_out, n_out);
+        auto src_view = tr_in(std::views::all(cont_in()));
+        auto out_view = tr_out(std::views::all(cont_exp()));
```

Comment:
It looks like there should be `cont_out` rather than `cont_exp`, so that `out_view` matches `cont_out`.

Reply:
Rather, `out_view` should be `exp_view`  named...